clion/src/main/kotlin/org/rust/clion/profiler/perf/RsPerfConfigurationExtension.kt

```diff
@@ -76,6 +78,13 @@ class RsPerfConfigurationExtension : CargoCommandConfigurationExtension() {
         ProfilerToolWindowManager.getInstance(project).addProfilerProcessTab(profilerProcess)
     }
 
+    private fun validatePerfSettings(project: Project): MisConfiguredException? {
+        val state = CPPProfilerSettings.instance.state
+        return validateLocalPath(state.executablePath.orEmpty(), "Perf executable", project, profilerConfigurable::class.java)
+            ?: validateFrequency(state.samplingFrequency, project, profilerConfigurable::class.java)
+            ?: validateOutputDirectory(state.outputDirectory.orEmpty(), project, profilerConfigurable::class.java)
+    }
```

Comment:
As I see, CLion has exactly the same check. Is it possible to have a single common check in some common code in CLion to avoid code duplication? Ideally, it would be great not to call this check at all in our plugin and rely on CLion validation but I'm not sure that it's easy

Reply:
That's a good idea, I've been going to reuse common code as much as possible in the future. This PR was originally planned as a hotfix so it's just a copy-paste (the same with the second review comment too)

Reply:
BTW, besides reusing common code, it's important to find out why CLion actually fails to detect `/usr/bin/perf` automatically, and I'm working on it as well